addon/data/shooter-interactive-worker.js

```diff
@@ -273,6 +281,9 @@ stateHandlers.crosshairs = {
   },
 
   mousedown: function (event) {
+    if (event.target.className === "pageshot-myshots") {
+      return;
+    }
```

Comment:
I originally added this in the isHeader method, but I couldn't tell if that was supposed to be only for Google Chrome. Is it?